modules/hostap/src/supp_api.c

```diff
@@ -443,6 +542,14 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 	uint8_t ssid_null_terminated[WIFI_SSID_MAX_LEN + 1];
 	uint8_t psk_null_terminated[WIFI_PSK_MAX_LEN + 1];
 	uint8_t sae_null_terminated[WIFI_SAE_PSWD_MAX_LEN + 1];
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
+	struct wifi_eap_cipher_config cipher_config = {NULL,   "DEFAULT:!EXP:!LOW", "CCMP",
+						       "CCMP", "AES-128-CMAC",      NULL};
+	char *method;
+	char phase1[256] = {0};
```

Comment:
Do we need 256 chars for this, just wanted to confirm because this seems quite long string.

Reply:
Yes, indeed, 256 bytes consume too many stack space, I checked supplicant code, 128 bytes to store phase1 info should enough.